.chglog/config.yml

```diff
@@ -0,0 +1,28 @@
+style: github
+template: CHANGELOG.tpl.md
+info:
+  title: CHANGELOG
+  repository_url: https://github.com/khauser/helm-charts
```

Comment:
This should probably not point to your fork.

Reply:
probably